### ISSUE TITLE
Consolidate different versions of the OWID logo that we use

### ISSUE THREAD

**Issue by marcelgerber** [owid/owid-grapher#2926]
## Description

We are using 4 different versions of the OWID logo at this point:
- The one used in grapher charts (recently updated)
- https://ourworldindata.org/images/published/default-featured-image.png, which is used as a fallback image for articles without a thumbnail
- https://ourworldindata.org/images/published/default-featured-image.png, which we're showing next to the license
- The one we use in the header, which is produced using plain HTML, and is using a slightly different font weight

You can see all of them being used on https://ourworldindata.org/human-development-index.

![image](https://github.com/owid/owid-grapher/assets/2641501/b629e0d6-7ac5-493a-898d-91ebc8a41df3)

**Comment by danyx23** [comment]
@mrwbkrm Which one is the version of the logo we should use? Is there even a single logo that is fit for purpose in all places?

**Comment by mrwbkrm** [comment]
@danyx23 When I worked on Grapher, I realized we needed to adapt the logo slightly depending on what size it was. That's also one of the reasons why I converted the text to shapes (which is something I think we should do to avoid getting undesired results in SVG exports, etc). Having said that, if it's easier to have the logo in the header as text, that's ok as it's visually consistent. 
In [this file](https://www.figma.com/file/NUWQ31ikHIc3BQnIJw53NK/%5BGrapher%5D-Library?type=design&node-id=4%3A113&mode=design&t=uFLy6nxirNRi7bQu-1) you can find 3 different sizes of the logo which I think should cover all of these use cases.
To recap I think the logo should be updated to the version linked above in the case of:
- thumbnails
- license
- And if we want to keep the logo as text in the header, to make sure that it is consistent with the logo linked above.
<img width="899" alt="Screenshot 2023-12-06 at 11 10 47" src="https://github.com/owid/owid-grapher/assets/113343623/4fcbb003-8b4f-4ed1-bf20-15316434c0f2">
